src/components/ContentWidgets/ToDoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+
+import React from 'react';
+import { ContentSingularData } from '../../_data/ContentSingularData';
+import './TodoStyle.css';
+
+
+export type TodoItem = {
+	name: string
+	check: string // checked | unchecked
+}
+
+// Return a unordered list of todos 
+//
+// Last Modified
+// Max Zhou
+// July 5, 2019
+//
+export const TodoList: React.FC<ContentSingularData> = ({ todoList_items }) => {
+  if (!todoList_items) {
+    return <></>
+	};
+	let elems = todoList_items.map((item: TodoItem, index: number) => <li className={item.check} key={index}>{item.name}</li>);
```

Comment:
Don't use the `index` as a key for react items. I know they taught it that way in class but it actually leads to performance downsides. Use a unique identifier like a generated hash or so instead.

Reply:
Also, don't create the variable here, just put it below where `{elems}` is.